ubuntudesign/www.ubuntu.com
# Add border to the bottom of the hero row

## Done
- Added a border to small screen hero section in mobile section

## QA
- Pull down this branch and run `make run`
- Go to http://127.0.0.1:8001/mobile
- Check that there is no border at the bottom of the hero section
- Scale the window down and see that the border appears
